libp2p/pubsub/gossipsub.py

```diff
@@ -215,44 +215,47 @@ def _get_peers_to_send(
         :param origin: peer id of the peer the message originate from.
         :return: a generator of the peer ids who we send data to.
         """
-        send_to: List[ID] = []
+        send_to: Set[ID] = set()
```

Comment:
IMO the return value of `_get_peers_to_send` can be changed to `Set[ID]` accordingly.